microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,6 +303,66 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
+
+To use this array, the following information is needed:
+
+* `snapshot.meta.location_fields`, to know how many fields each location group has.
+* `snapshot.nodes`, to link a location group back to a node.
+
+The current `snapshot.meta.location_fields` has the information for each location group:
+| Index in location_fields | Name | Description |
+| --- | --- | --- |
+| `0` | `object_index` | The index of the associated node in `snapshot.nodes` array. |
+| `1` | `script_id` | The id of the script that creates the node which `object_index` is pointing to. |
+| `2` | `line` | The line number in the script of the node creation. |
+| `3` | `column` | The column number in the script of the node creation. |
+
+The following code example shows how to link the `snapshot.locations` Array to `snapshot.nodes` Array.
```

Comment:
```suggestion
The following code example shows how to link the `snapshot.locations` array to the `snapshot.nodes` array.
```
if we write capital A "Array", it would probably need backticks as code/type.  Unless special convention in this article.  Need to scope out the rendered page decide style.  Lowercase should work perfectly clearly.  Options:

The following code example shows how to link the `snapshot.locations` array to the `snapshot.nodes` array.

The following code example shows how to link the `snapshot.locations` `Array` to the `snapshot.nodes` `Array`.